openmdao/recorders/sqlite_recorder.py

```diff
@@ -26,18 +25,20 @@
 
 
 """
-SQL case output format version history.
----------------------------------------
+SQL case database version history.
+----------------------------------
+5 -- OpenMDAO 2.5
```

Comment:
There is so much here that I can't really tell, but did you add a test that verifies we can read a version 4 recording?

Reply:
Good point... I did not.   Will cook up a new test.

Reply:
done